english/your-dog/your-dog.md

```diff
@@ -3,4 +3,6 @@ Then you wake up! Or did you really? But your dog is licking you in the face. Lo
 An evil vile smell of rotten, no desicated, dry dog food that would haunt an arch demon used to the
 hellish smell of sulfur fire burning flesh.
 
-Then your dog speaks to you and says ["follow me ..."](../play-with-dog/dog.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.